packages/amplication-git-utils/src/git/git.module.ts

```diff
@@ -13,7 +12,7 @@ import { SERVICE_NAME } from "./git.constants";
       metadata: { service: SERVICE_NAME },
     }),
   ],
-  providers: [GitService, GitServiceFactory, GithubService],
-  exports: [GitService, GitServiceFactory, GithubService],
+  providers: [GitFactory, GithubService],
+  exports: [GitFactory, GithubService],
```

Comment:
Since we use a Class Factory now, we don't need to inject this class through DI.

```suggestion
  providers: [GitFactory],
  exports: [GitFactory],
```